src/nrnmpi/mpispike.cpp

```diff
@@ -93,6 +95,7 @@ static void make_spikebuf_type() {
 #endif
 
 int nrnmpi_spike_exchange() {
+  nrn::Instrumentor::phase_begin("spike-exchange");
```

Comment:
How about:
```suggestion
  nrn::Instrumentor::phase_begin("nrnmpi_spike_exchange");
```
easier to x-ref? 

same goes for the ones after.

Reply:
I stuck to having the same naming as in coreneuron (this will allow us to use the same post-proc scripts for both). But I'm open to change the labels. @pramodk , any preference?

Reply:
For the reasons mentioned above, we should use the same phase name. At the top level (e.g. for psolve step), we can have different name between neuron and coreneuron.

I will setup this locally and look at the profile reports shortly.